apps/nextra/pages/zh/network/nodes/validator-node/connect-nodes/staking-pool-operations.mdx

```diff
@@ -112,7 +112,7 @@ aptos stake withdraw-stake \
 ### 更新佣金
 
 ```bash filename="Terminal"
-aptos move run --function-id 0x1::staking_contract::update_commision \
+aptos move run --function-id 0x1::staking_contract::update_commission \
```

Comment:
Unfortunately, it's actually only a single s right now https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/staking_contract.move#L494-L498